pkgs/applications/science/astronomy/gildas/default.nix

```diff
@@ -0,0 +1,55 @@
+{ stdenv, fetchurl, gtk2 , pkgconfig , python27 , gfortran , python27Packages , lesstif , cfitsio , getopt , perl , groff }:
+
+stdenv.mkDerivation rec {
+  version = "feb17d";
```

Comment:
I agree that this can break version sorting, but I'm not responsible for the versioning of Gildas... I'll propose a different version number for the nix package so...